### ISSUE TITLE
Update frontend .env / config 

### ISSUE THREAD

**Issue by robinv85** [FightPandemics/FightPandemics#468]
For the ESLint config with Create-React-App there was a recent adjustement to add custom ESLint rules, which requires `EXTEND_ESLINT=true` in the `client/.env` file.

This file is not in added in version control since it's in `.gitignore` because it contains `REACT_APP_AIRTABLE_API_KEY` and `REACT_APP_AIRTABLE_BASE`. 

This might cause issues if newcomers setup the repository and forget to add this variable.

While these 2 settisng will probably be removed eventually and replaced by our own mongo, we can move them to `client/config.js` and add `client/.env` to version control to use for the CRA setting. Alt

**Comment by idjevm** [comment]
@robinv85 Do you want to add specifically the .env file? 

I noticed that the `.env.example` file is already included there like this ->
![image](https://user-images.githubusercontent.com/25408660/83872186-7c452900-a6ff-11ea-8078-47f90ed839b7.png)

 Perhaps add somewhere on Eng onboarding to request for the Airtable API Key and the baseId. or I can make an update to just have the baseId, and have new Engs request the Airtable API KEY if it is better?

**Comment by robinv85** [comment]
@idjevm for now, these are in `.env` but airtable will be removed from the frontend eventually
otherwise we could move them to the `client/config.js` too, but i think it's not needed
so the only thing really needed here is when we remove airtable just add `.env` to git and then the problem is solved i think :)
I don't know if there is a ticket to remove airtable or where it's used currently, if you want you can work on that too

**Comment by robinv85** [comment]
see #342 i think that depends on geolocation backend update first which is now highest priority

**Comment by vikjoshi** [comment]
@robinv85 @idjevm - from our product/engineering meeting today: we need to add feature flags so we can hide/unhide certain features if they are not ready for launch. Robin's comment on how to do that: 
"i wonder, feature flags always need the .env file i guess? because that ESLint stuff for frontend would require to put the .env file in git so might be related"

**Comment by robinv85** [comment]
yeah I don't see another way, I think @mannykary also suggested we could use multiple `.env` files so for example one that we add to git and one that can contain things like keys. but in the frontend I don't think we will need that if we remove the airtable keys here.
Only other configs is currently in `config.js` the password min and max length but that isn't really needed either I think

**Comment by joshmorel** [comment]
This will be resolved with #649

**Comment by joshmorel** [comment]
Will be resolved by #649

**Comment by robinv85** [comment]
@vikjoshi and @joshmorel If I understand then this issue is resolved so the `/client/.env` is now in git version control
This doesn't provide feature flags functionality, but if we add that functionality, it will probably use this `.env` file to set them, at least for development
Also I think @MohamedEl1 asked about setting the GTM id in the frontend, I suppose this is also related, not sure if we want to add that in `.env` which is in git?

**Comment by mannykary** [comment]
I think we should not add the GTM key (or any other secrets) to the `.env` file committed to git. We should store secrets/keys used by the client container either Github Secrets (in repo settings), or in an encrypted S3 bucket. Since we don't have a lot of secrets for the client, we can store them in Github Secrets for now. We would just pull the keys from Github Secrets and append them to the `.env` file while building the container.

**Comment by joshmorel** [comment]
@robinv85 I see now this is only partially resolved. For some reason I missed the part not related to the `REACT_APP_AIRTABLE*` stuff. These are all removed from F/E resolved part.

But committing the `client/.env` file with `ESLINT=true` - that wasn't done it's still in `.gitignore`. Is the reason we want to commit because people aren't always copying the `.env.example` file? Could we add a `RUN` command into `Dockerfile.env` to add this line to .env if it doesn't exist (because there may be other stuff in .env like GTM we don't want to commit to git).
